docs/introduction/get_involved.rst

```diff
@@ -0,0 +1,75 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2024 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+.. _introduction_get_involved:
+
+Get involved
+============
+
+How to get in contact with SCORE
```

Comment:
We have a mixture of S-CORE and SCORE in the documentation. We need to align on one way to write it as default.
See issue #663